client/cmd/dexcctl/config.go

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
```

Comment:
Is it nearly verbatim copy from dcrctl's config.go?  Otherwise we should strive to keep a consistent (blueoak) license throughout where possible.  If files are really taken from (not just based on) dcrctl, technically we need to get perm from @davecgh and whoever worked on those files to change the license.  An unfortunately hiccup with our decision to change the license...

Reply:
The configure function is quite different. The helper function to clean the path to a cert is verbatim. So, maybe half original. I wasn't thinking about the license at all. Sorry about that. I'll change everything to the blue oak header if that's ok. I really don't know what's right here, though.

Reply:
I don't know what's right either.  But I'm pretty sure we can't change the license on the copied files without permission from the license holders, even though the licenses are compatible.  Anything that's essentially new can be the new license, other stuff can stay until we figure out the proper way to change things.

Reply:
None are exact copies, while there are similarities. Changed all to blue oak. Or is it blueoak?